symphco/symph-libs
# [Chore][Tony] Update README with Logger Installation Instructions

## What does this PR do?
- Adds extensive instructions for setting up and installing the project's logger package locally through `npm link`.

## Why did you choose this solution?
- To provide clear steps for developers to test the logger package in their local development environments before publishing.

## What is the dependency graph of the changes?
- README.md modifications
- Additional documentation related to npm package linking

## How to test/verify?
- Follow the newly added instructions in the README to install the `logger` locally.
- Verify the local installation by linking the package to another project and importing it as described.

## Related ticket url:
-
